universal-application-tool-0.0.1/app/views/questiontypes/NumberQuestionRenderer.java

```diff
@@ -23,9 +24,19 @@ public NumberQuestionRenderer(ApplicantQuestion question) {
   public Tag render() {
     NumberQuestion numberQuestion = question.createNumberQuestion();
 
+    FieldWithLabel numberField =
+        FieldWithLabel.number()
+            .setFieldName(numberQuestion.getNumberPath().path())
+            .setFloatLabel(true);
+    if (numberQuestion.getNumberValue().isPresent()) {
+      // TODO: [Refactor] Oof! Converting Optional<Long> to OptionalLong.
+      OptionalLong value = OptionalLong.of(numberQuestion.getNumberValue().orElse(0L));
```

Comment:
Suggestion (if-minor): Please finish the refactor if this doesn't take more than 1 hour of work.